sim/warrior/arms/mortal_strike.go

```diff
@@ -35,7 +35,7 @@ func (war *ArmsWarrior) RegisterMortalStrike() {
 		CritMultiplier:   war.DefaultMeleeCritMultiplier(),
 
 		ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
-			baseDamage := 423 + 0.8*(spell.Unit.MHNormalizedWeaponDamage(sim, spell.MeleeAttackPower()))
+			baseDamage := 634 + 1.5*(spell.Unit.MHNormalizedWeaponDamage(sim, spell.MeleeAttackPower()))
```

Comment:
The 1.5 multiplier should also be applied to the base damage. How we normally model weapon skills is to put the multiplier in the spells `DamageMultiplier` field. The base damage should be the exact same base that you get for this spell using the `Class Base Scaling * Coefficient`. Mortal Strikes Coeff shows as `0.37599998713` and warriors base scaling value you can get from `core.GetClassSpellScalingCoefficient` it comes out at 423 at level 85 which is the correct value it had before your update